Sources/Turf/Geometries/LineString.swift

```diff
@@ -308,4 +308,25 @@ extension LineString {
         // Ported from https://github.com/Turfjs/turf/blob/4e8342acb1dbd099f5e91c8ee27f05fb2647ee1b/packages/turf-simplify/lib/simplify.js
         coordinates = Simplifier.simplify(coordinates, tolerance: tolerance, highestQuality: highestQuality)
     }
+    
+    /**
+     Returns all intersections with another `LineString`.
+     
+     This function is roughly equivalent to the [turf-line-intersect](https://turfjs.org/docs/#lineIntersect) package of Turf.js ([source code](https://github.com/Turfjs/turf/tree/master/packages/turf-line-intersect/)).
```

Comment:
Mention `intersection(_:_:)` as an alternative for working with individual line segments.

Update the correspondence table in the readme to link to this method:

https://github.com/mapbox/turf-swift/blob/1806393ee69e219ddc30ebf16cab3afc0436fe41/README.md#L90